tests/cpp/test_index_select.cpp

```diff
@@ -857,4 +857,51 @@ TEST_F(NVFuserTest, IndexSelectVectorization3DCase1) {
   testValidate(&fusion, outputs, {t0, t1}, __LINE__, __FILE__);
 }
 
+TEST_F(NVFuserTest, IndexPutAccumulate) {
+  int64_t vocab_size = 1024;
+  int64_t hidden_size = 3584;
+  int64_t seq_size = 3000;
+  // Note: The semantics doesn't support broadcast on operands, adding `size 1`
+  // check just to ensure the ID mapping is done correctly.
```

Comment:
How is it ensured? Does just running the test exercise the mapping well?

Reply:
I think so. Since earlier without fixing the mapping, the test hits an assert.

Reply:
Which assert are you referring to? 

Why not explicitly testing the result of `PairwiseLogicalDomainMap`?

Reply:
explicit test sounds good. Will switch to that.